### DISCUSSION TITLE
[Question]: 0.14 logins, do I have to enable something to make them work?

### DISCUSSION THREAD

**Discussion by superdong69** [blakeblackshear/frigate#11635] [Ask A Question] [answered]
### What is your question:

When I navigate to my frigate URL on any computer on the lan, you have full access to the server. I changed the admin password. But even when I login with that account, Current User: says anonymous. I'd like to lock down the UI, and only have an account for myself and my wife to view the system. I'm guessing I just need to enable a flag somewhere, but can't for the life of me find it.

**Comment by NickM-27** [comment, marked as answer]
there is no flag, just need to use the correct port. Port 8080 of the container is authenticated and port 5000 is unauthenticated

**Reply by superdong69** [reply]
Anyway to shut down the unauthenticated port option?

**Reply by NickM-27** [reply]
don't map port 5000 out of the container. But keep in mind currently the home assistant integration does not support authentication. The docs explain ways to work with this by having local-only access https://deploy-preview-11419--frigate-docs.netlify.app/integrations/home-assistant#docker-compose-examples

**Reply by superdong69** [reply]
Good point. I'll hold off on that. In the off chance that anyone stumbles upon this, and you can't access frigate on 8080, don't forget to map the port in your docker-compose.yaml!

**Reply by Mocar2D** [reply]
@NickM-27 is there an authenticated port in the home assistant frigate add-on? Since in the config of it in hassos i can specify which port to use for the web interface, but no matter what i did port 8080 is not open

**Reply by NickM-27** [reply]
ah good point, we need to add that to the addon config

**Reply by RichardUUU** [reply]
Is there a container variable we can use to change 8080 to some other port?

**Reply by NickM-27** [reply]
it's not 8080 in the final release, it is 8971. and just like any other port, you change it in the docker config

**Reply by RichardUUU** [reply]
Did you mean 8971 ? That's what's shown at https://docs.frigate.video/frigate/installation
I'm using a cloudflare tunnel which would normally route to an internal http address e.g. http://192.168.1.123:8971 but that gives me an HTTPS error.

**Reply by NickM-27** [reply]
it gives you an https error because cloudflare is providing tls so you need to disable tls in frigate
